src/extension.ts

```diff
@@ -169,6 +171,9 @@ async function customizeAsync(node: ClusterExplorerV1.ClusterExplorerResourceNod
     if (node.nodeType === 'resource' && node.resourceKind.manifestKind === 'BuildConfig') {
         treeItem.collapsibleState = vscode.TreeItemCollapsibleState.Collapsed;
     }
+    if (node.nodeType === 'resource' && node.resourceKind.manifestKind === 'DeploymentConfig') {
```

Comment:
no need to repeat if with different kind of manifest condition, just add `node.resourceKind.manifestKind === 'DeploymentConfig'` condition to previous if statement.
``` JavaScript
if (node.nodeType === 'resource' && (node.resourceKind.manifestKind === 'BuildConfig' || node.resourceKind.manifestKind === 'DeploymentConfig')) {
    treeItem.collapsibleState = vscode.TreeItemCollapsibleState.Collapsed;
}
```